docs/v0.5/search/index.html

```diff
@@ -1029,6 +1029,18 @@ <h1></h1>
anizations seeking to adopt and operate a service mesh or collection of service meshes.\n\nDesign Goals\n\nThe designs in this specification should result in enabling:\n\n\n  \n    User experience should be intuitive\n\n    This is achieved through sensible defaults, consistency of user interaction paradigms and features that delight the user.\n  \n  \n    Meshery UI should simplify user management of workloads and service meshes\n\n    Meshery UI needs to be simple, but powerful. This is achieved through intuitive layouts and predefined filters that accommodate common tasks.\n  \n  \n    Extensible\n\n    Meshery UI should be a first-class component of Meshery, but also facilitate third-party integrations.\n  \n\n\nDesign Objectives\n\nThe designs in this specification should result in enabling:\n\n\n  Meshery UI should be event-driven where possible.\n\n\nSetup\n\nLinting-UI\n\n\n  \n    When contributing to this project, it is advisable to\n\n    \n      \n        Use eslint plugin for Visual Studio Code.\n      \n      \n        Disable plugins other than eslint for formatting and linting, if any.\n      \n    \n  \n\n\nInstall UI dependencies\n\nTo install/update the UI dependencies:\n\nmake ui-setup\n\n\nBuild and export UI\n\nTo build and export the UI code:\n\nmake ui-build\n\n\n\n  Now that the UI code is built, Meshery UI will be available at http://localhost:9081 when Meshery Server is running (Read below).\n\n\n\n  Changes are not recompiled directly, you will have to run to rebuild the UI to see them\n\n\nRun Meshery Server\n\nTo start running Meshery Server locally:\n\n$ make server\n\n\n  Now, Meshery will run on the default port http://localhost:9081.\n\n\n\nUsage of 'make run-fast' is deprecated!\n\n\n\nUI Development Server\n\nIf you want to work on the UI, it will be a good idea to use the included UI development server. You can run the UI development server by running the following command:\n\nmake ui\n\n\nRefer to Contributing to Meshery Server, if needed.\n\n\n  Make sure to have Meshery Server configured, up and running on the default port http://localhost:9081 before proceeding to access and work on the UI server at http://localhost:3000.\n\n\n\n  Any UI changes made now will automatically be recompiled and served in the browser.\n\n\nRunning Cypress integration tests\n\nTo run cypress integration tests, a convenience make target called ui-integration-tests that installs dependencies in /ui and /provider-ui folders as prerequisite and invokes ci-test-integration npm script found in /ui/package.json\n\n   \n     $ make ui-integration-tests\n   \n\nAbove command must be run from Meshery repository's root folder.\n\n\n\nRefer to Meshery Cypress Testing for details of how to contribute and benefit from Meshery Cypress (integration &amp; end-to-end) testing.\n\nRunning Meshery from IDE\n\nAll of the above steps would get the Meshery’s development server running for you to work on in any IDE of your choice.\n\nSuggested Reading"
```

Comment:
@Ambareen09, let's forego updates to older versions of the Meshery Docs as they are snapshots in time, frozen at the time of final release of that version.

Reply:
you mean, I should revert this change?

Reply:
Oh sorry, I didn't realize that. Made the suggested change.